terminus-ui/chip/src/chip.component.scss

```diff
@@ -4,38 +4,16 @@
 @import './../../scss/helpers/spacing';
 @import './../../scss/helpers/typography';
 
-$ts-chip-height: 35px;
-$ts-chip-vertical-padding: 3px;
-$ts-chip-horizontal-padding: 8px;
-$ts-chip-remove-width: 18px;
-$ts-chip-remove-height: 18px;
+
 $ts-chip-focus-background-color: color(utility);
-$ts-chip-color: color(pure, dark);
-$c-chip-top-margin: 7px;
-$c-chip-right-margin: 4px;
-$c-chip-padding: 25px;
 
 .ts-chip {
```

Comment:
I was having a weird layout issue when inside a flex layout. Moving most styles to the c-chip class solved it for me

Reply:
yeah, chip list didn't wrap originally - it just expanded past the limit. I think the CSS I added to fix that introduced this issue